pose_calculator.py

```diff
@@ -143,30 +149,54 @@ def get_values(self, units="m", display=True):
             return [None, None, None], [None, None, None]
 
         if len(contours) < 1:
-            self.display_windows(frame, mask)
+            if display:
+                self.display_windows(frame, mask)
             return [None, None, None], [None, None, None]
 
         # sort contours by area in descending order
         contours.sort(key=lambda c: cv2.contourArea(c), reverse=True)
         c = contours[0]  # use the largest contour
 
         if cv2.contourArea(c) < 100:
-            self.display_windows(frame, mask)
+            if display:
+                self.display_windows(frame, mask)
             return [None, None, None], [None, None, None]
 
         corners = self.get_corners(c)
         area = cv2.contourArea(c)
         cv2.drawContours(frame, [corners], -1, (0, 0, 255), 2)
 
-        rvec, tvec = self.get_angle_dist(Target(corners, area))
+        rvec, tvec, ret = self.get_angle_dist(Target(corners, area))
+        if not ret:
+            if display:
+                self.display_windows(frame, mask)
+            return [None, None, None], [None, None, None]
         rmat, _ = cv2.Rodrigues(rvec)  # convert rotation vector to matrix
 
-        # rx, ry, rz = self.get_euler_from_rodrigues(rmat)
-        r_euler = R.from_rotvec(rvec.flatten()).as_euler('xyz', degrees=True)
-        rx, ry, rz = r_euler[0], r_euler[1], r_euler[2]
+        rx, ry, rz = self.get_euler_from_rodrigues(rmat)
```

Comment:
Is there a reason we swaped back to using the rodrigues matrix? If so we should remove the scipy import